app/bundles/ApiBundle/Controller/CommonApiController.php

```diff
@@ -861,6 +862,22 @@ protected function getEntityDefaultProperties($entity)
         return $properties;
     }
 
+    protected function sanitizeProperties(object $entity, array $parameters)
```

Comment:
I think this shouldn't be located directly in the CommonApiController as all entities use it. But only 2 (contact, company) have custom fields. How about to implement it into https://github.com/mautic/mautic/blob/features/app/bundles/LeadBundle/Controller/Api/CustomFieldsApiControllerTrait.php ?